src/ophyd_async/epics/eiger/_odin_io.py

```diff
@@ -80,9 +81,11 @@ def __init__(
         self,
         path_provider: PathProvider,
         odin_driver: Odin,
+        eiger_bit_depth: SignalR[int],
```

Comment:
Should: This needs to be a reference (https://blueskyproject.io/ophyd-async/main/_api/ophyd_async/ophyd_async.core.html#ophyd_async.core.Reference) so that it doesn't get renamed

Reply:
FYI https://github.com/bluesky/ophyd-async/issues/871